addons/registry/2.7.1/tmpl-deployment-objectstore.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: registry-config
+  labels:
+    app: registry
+data:
+  config.yml: |-
+    health:
+      storagedriver:
+        enabled: true
+        interval: 10s
+        threshold: 3
+    auth:
+      htpasswd:
+        realm: basic-realm
+        path: /auth/htpasswd
+    http:
+      addr: :443
+      headers:
+        X-Content-Type-Options:
+        - nosniff
+      tls:
+        certificate: /etc/pki/registry.crt
+        key: /etc/pki/registry.key
+    log:
+      fields:
+        service: registry
+    storage:
+      s3:
+        region: "us-east-1"
+        regionendpoint: http://$OBJECT_STORE_CLUSTER_IP
+        bucket: docker-registry
+        accesskey: $OBJECT_STORE_ACCESS_KEY
+        secretkey: $OBJECT_STORE_SECRET_KEY
```

Comment:
should this be a secret? this is how we do it in aka

Reply:
Probably. You should be able to reuse the `kotsadm-s3` secret for that 1 env var.